modules/monitor/hardware/gps/gps_monitor.cc

```diff
@@ -24,6 +26,13 @@ DEFINE_string(gps_hardware_name, "GPS", "Name of the GPS hardware.");
 DEFINE_string(gps_monitor_name, "GpsMonitor", "Name of the GPS monitor.");
 DEFINE_double(gps_monitor_interval, 3, "GPS status checking interval (s).");
 
+DEFINE_double(acceptable_gnss_best_pose_std_dev, 0.5,
+              "Acceptable Gnss BestPose standard deviation on latitude, "
+              "longitude and height.");
+
+DEFINE_double(acceptable_gnss_best_pose_unstable_duration, 5,
```

Comment:
what is the unit of this duration ? Could you add it in the description?

Reply:
Done.